app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,144 @@
-// Code your JavaScript / jQuery solution here
+var winningCombos = [
```

Comment:
As these will never change, it is best to declare this as a constant: `const WINNING_COMBOS`